sdk/servicebus/azure-messaging-servicebus/src/main/java/com/azure/messaging/servicebus/implementation/ServiceBusManagementSerializer.java

```diff
@@ -56,14 +59,23 @@ public String serialize(Object object, SerializerEncoding encoding) throws IOExc
         }
 
         final String namespace = namespaceMatcher.group("namespace");
-        final String replaced = contents
+        String replaced = contents
             .replaceAll(namespace + ":", "")
             .replace("xmlns:" + namespace + "=", "xmlns=");
 
         if (!CreateRuleBody.class.equals(clazz)) {
             return replaced;
         }
 
+        // This hack is here because value of custom property within RuleFilter should have a namespace like xmlns:d6p1="http://www.w3.org/2001/XMLSchema" ns0:type="d6p1:string".
+        final Matcher filterValue = FILTER_VALUE_PATTERN.matcher(replaced);
```

Comment:
Does the generated xml string have the value of `RULE_VALUE_ATTRIBUTE_XML `?

Reply:
Better put this new code logic in a `if` statement because this applies to only RuleFilter

Reply:
According to your Comment, fixed in new version.